### DISCUSSION TITLE
@Mike777Khoo 已能登录 谢谢

### DISCUSSION THREAD

**Discussion by xwyqsl** [getlantern/lantern-forum-cn#863] [General]

**Comment by MaxMKMax** [comment]
@xwyqsl 您好. 因为专业版账号升级系统有问题了. 所以没有被升级成功. 我这里帮您提交问题来让他们帮您升级好吗? 您不要退款好不好? 谢谢您! 非常抱歉影响了您的购买和使用体验.

**Comment by greySkiesMarsh** [comment]
Hi there @xwyqsl, this is Marsh, the support lead here at Lantern. 

It sounds like you purchased your plan through a 3rd party seller, which unfortunately means that we don't have the ability to refund your purchase directly. 

However, if you can let me know which plan you purchased, I can definitely get your account fixed and will add a complimentary month of Pro access for your troubles.

**Reply by xwyqsl** [reply]
@greySkiesMarsh 
I bought a 2-year plan on the 11th, but I can't log in and no one has contacted me.
 Please help me solve it or refund it, thank you.

**Reply by greySkiesMarsh** [reply]
> @greySkiesMarsh I bought a 2-year plan on the 11th, but I can't log in and no one has contacted me. I tried to apply for the verification code myself via this account [jewel_sisi@126.com](mailto:jewel_sisi@126.com),but I didn't receive the verification code. I tried to change my email address but I don't know how to do it. Anyway, I now need you to send me a verification code to the new account [jewel_sisi@hotmail.com](mailto:jewel_sisi@hotmail.com) . Please help me solve it or refund it, thank you.

Hi again. I apologize for the delay in getting this resolved, but your account should be fully functional as of the time of this reply. I've included 3 complimentary months of Pro access for your trouble.

**Reply by xwyqsl** [reply]
My account has been successfully logged in, thank you. 
However, I noticed that the email form now is blank. Do I need to provide an email address for my account?
![截屏2024-09-24 06 08 04](https://github.com/user-attachments/assets/f004ad84-b4b0-4280-add0-527ad7c3db5d)

**Comment by MaxMKMax** [comment]
@xwyqsl 我也帮您提醒支付团队了.

**Comment by MaxMKMax** [comment]
@xwyqsl Yes, you may reupdate with your new email that you wish to have in the account.
